eras/conway/impl/src/Cardano/Ledger/Conway/PParams.hs

```diff
@@ -840,34 +840,34 @@ conwayPParamsPairs pp =
 
 instance Era era => FromJSON (ConwayPParams Identity era) where
   parseJSON =
-    withObject "PParams" $ \obj ->
+    withObject "ProtocolParameters" $ \obj ->
       ConwayPParams
-        <$> obj .: "minFeeA"
-        <*> obj .: "minFeeB"
+        <$> obj .: "txFeePerByte"
+        <*> obj .: "txFeeFixed"
         <*> obj .: "maxBlockBodySize"
         <*> obj .: "maxTxSize"
         <*> obj .: "maxBlockHeaderSize"
-        <*> obj .: "keyDeposit"
-        <*> obj .: "poolDeposit"
-        <*> obj .: "eMax"
-        <*> obj .: "nOpt"
-        <*> obj .: "a0"
-        <*> obj .: "rho"
-        <*> obj .: "tau"
+        <*> obj .: "stakeAddressDeposit"
+        <*> obj .: "stakePoolDeposit"
+        <*> obj .: "poolRetireMaxEpoch"
+        <*> obj .: "stakePoolTargetNum"
+        <*> obj .: "poolPledgeInfluence"
+        <*> obj .: "monetaryExpansion"
+        <*> obj .: "treasuryCut"
         <*> obj .: "protocolVersion"
         <*> obj .: "minPoolCost" .!= mempty
-        <*> obj .: "coinsPerUTxOByte"
-        <*> obj .: "costmdls"
-        <*> obj .: "prices"
-        <*> obj .: "maxTxExUnits"
-        <*> obj .: "maxBlockExUnits"
-        <*> obj .: "maxValSize"
+        <*> obj .: "utxoCostPerByte"
+        <*> obj .: "costModels"
+        <*> obj .: "executionUnitPrices"
+        <*> obj .: "maxTxExecutionUnits"
+        <*> obj .: "maxBlockExecutionUnits"
+        <*> obj .: "maxValueSize"
         <*> obj .: "collateralPercentage"
         <*> obj .: "maxCollateralInputs"
         <*> obj .: "poolVotingThresholds"
         <*> obj .: "dRepVotingThresholds"
-        <*> obj .: "committeeMinSize"
-        <*> obj .: "committeeMaxTermLength"
+        <*> obj .: "minCommitteeSize"
+        <*> obj .: "committeeTermLength"
```

Comment:
These are new parameters, so we don't have problems with backwards compatibility yet. I would suggest naming them the same way they are in the spec, because the names from cardano-api are quite inconsistent (I believe they reflect the original names that we had that have been improved since then).
```suggestion
        <*> obj .: "committeeMinSize"
        <*> obj .: "committeeMaxTermLength"
```